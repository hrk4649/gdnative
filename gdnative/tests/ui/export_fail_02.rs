use gdnative::prelude::*;

#[derive(Export, ToVariant)]
pub struct Foo {
    bar: i32,
}

fn main() {}
