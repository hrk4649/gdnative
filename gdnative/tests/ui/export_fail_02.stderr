error: #[derive(Export)] only supports fieldless enums
 --> tests/ui/export_fail_02.rs:4:5
  |
4 | pub struct Foo {
  |     ^^^^^^
